hangman.php

```diff
@@ -0,0 +1,88 @@
+<?php
+    $word="";
+    function getword()
+    {
+        $file="BandsList.txt";
+        $fopen = fopen($file, "r");
+        $fread = fread($fopen,filesize($file));
+        $split = explode("\n", $fread);
+        $array[] = null;
+        foreach ($split as $string)
+        {
+            array_push($array,$string);
+        }
+        $word_index = array_rand($array,1);
+        $word = $array[$word_index];
+        return $word;
+    } 
+    function setBlanks()
+    {
+        global $word;
+       $word=getword(); 
+       $no_of_blanks=strlen($word);
+       for($i=1;$i<=$no_of_blanks;$i++)
+       {
+            echo "<input type = text size='1'  id=".$i." readonly></input>";
+       }
+    }
+?>
+
+
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Hangman Game</title>
+    <h1> HANGMAN </h1>
+</head>
+<body>
+    <b>Guess The Band name</b>
+    <?php setblanks()?>
+    <?php
```

Comment:
Don't open and close the PHP tags again and again. Open it once, write the PHP code and then close it. Follow this practice everywhere.